### PR TITLE
[ADD] : Added workshop 16 (Rust-Tetra) 

### PR DESCRIPTION
**Pull request by nicolasheude**
This Pull-request contains the new Rust workshop, which aims to create a video game with the Tetra framework. So we add the README, the SETUP as well as some useful resources for the realization of the workshop.

### CODE REVIEW

File: software/16.Rust-Tetra/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+> :exclamation: We strongly advise you to use the resources given for this exercise.
+
+First, you need an image. Make sure you have a `PoC.png` file in the `assets/images` folder.
+
+Your program must display an image with the parameter :
+- The scale.
+
+- The postition.
+
+- The origin of the image.
```
Comment (TomChv) [inline software/16.Rust-Tetra/README.md:73]: Add types

File: software/16.Rust-Tetra/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+With the previous step, you have displayed your first window. Now, why not display some text in this window?
+
+Using the font provided with the workshop, the file `DejaVuSansMono.ttf` must be in a folder `/assets/font/`.
+
+To display text, you need to follow three steps:
+
+1. Add the font to your program.
+
+2. Create a variable (a vector) called `vector_text` with the value `Hello world!` and the font of your assets.
```
Comment (TomChv) [inline software/16.Rust-Tetra/README.md:50]:
```suggestion
2. Create a variable `vector_text` of type `vector` with the value `Hello world!` and the font of your assets.
```

**Review by TomChv — CHANGES_REQUESTED** [review]
LGTM, great job dude !
Just minor change

File: software/16.Rust-Tetra/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+ - [The Rust website](https://www.rust-lang.org/fr)
+ - [Tour of Rust](https://tourofrust.com/00_en.html)
+
+## Authors
+- [Yoel EDERY](https://www.linkedin.com/in/yoel-edery-957117210/)
+- [Nicolas HEUDE](https://www.linkedin.com/in/nicolas-heude-525567197)
+
+## Organization
+
+- [📒 Linkedin](https://www.linkedin.com/company/pocinnovation/mycompany/)
```
Comment (PaulMonnery) [inline software/16.Rust-Tetra/README.md:150]: `mycompany` in the url is not needed I think :thinking: 
```suggestion
- [📒 Linkedin](https://www.linkedin.com/company/pocinnovation/mycompany/)
- [📒 Linkedin](https://www.linkedin.com/company/pocinnovation/)
```

**Pull request merged**